examples/bazel_managed_deps/deps.spec.js

```diff
@@ -10,4 +10,15 @@ describe('dependencies', () => {
      () => {
        require('jasmine-core');
      });
+
+  it('.bin files should be in runfiles via @npm//:bin_files data dep', () => {
+    try {
+      expect(require.resolve('.bin/jasmine').endsWith('npm/node_modules/.bin/jasmine')).toBe(true);
+      expect(require.resolve('.bin/tsc').endsWith('npm/node_modules/.bin/tsc')).toBe(true);
+      expect(require.resolve('.bin/tsserver').endsWith('npm/node_modules/.bin/tsserver'))
+          .toBe(true);
+    } catch (_) {
+      fail('.bin file not resolved');
```

Comment:
nit: should say which require failed, or just make three assertions

Reply:
👍